### ISSUE TITLE
[Bottom Navigation] RN 0.42.0 Not full width

### ISSUE THREAD

**Issue by iamkyleee** [xotahal/react-native-material-ui#111]
![screenshot_20170305-051505](https://cloud.githubusercontent.com/assets/7822506/23587511/df565eba-0162-11e7-98ee-96a9dcef79ba.png)

**Comment by brianfoody** [comment]
The issue stems from the default minWidth variable in src/styles/getTheme.js. I've logged an issue with React Native here. You can fork and remove the minWidth as a temp workaround.

https://github.com/facebook/react-native/issues/12878

**Comment by HofmannZ** [comment]
Yeah seems to be an issue with react-native itself, worked all fine when I build the component back on 0.40.0
